CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Braintree Android SDK Release Notes
 
 ## unreleased
+* PayPalNativeCheckout (BETA)
+  * Fixes a bug where an error was not thrown inside `PayPalNativeCheckoutClient`
```

Comment:
```suggestion
* PayPalNativeCheckout (BETA)

  * Fixes a bug where an error was not thrown inside `PayPalNativeCheckoutClient` when no PayPal response was received from the API
```
^ I wonder if we can make this a bit more explicit as to when we weren't handling the error - as it is now ready a bit like we were just never handling errors in the client which I don't believe is the case